modules/hcp-migrate-aws-multiarch-nodepools.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-manage/hcp-manage-aws.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-migrate-aws-multiarch-nodepools_{context}"]
+= Creating node pools on the multi-architecture hosted cluster
+
+After transitioning your hosted cluster from single-architecture to multi-architecture, you must create node pools manually. You can create node pools on compute machines based on 64-bit AMD and 64-bit ARM architectures.
```

Comment:
> you must create node pools manually.

This sounds a little funny to me. Isn't this true even when using single-architecture after the HC has been created?